.kuzzlerc.sample

```diff
@@ -585,6 +585,10 @@
           }
         }
       },
+      // Maximum number of simultaneous scroll session
+      "maxScroll": 300,
```

Comment:
This parameter is already configurable directly in Elasticsearch. Moreover, IMHO, it is not the role of Kuzzle to manage this.

Reply:
Yes I know but it's more convenient to configure this value inside Kuzzle 
configuration file than in Elasticsearch.  

Actually we could also restrict the `maxScrollDuration` directly with ES but I see at least 2 reasons to keep theses restrictions inside Kuzzle:
 - less links between ES and Kuzzle: Kuzzle handle everything
 - better error message: ES errors are known to be a mess

Reply:
What happens if the value set in that configuration is greater than the elasticsearch one?

Reply:
That's why I'd prefer we leave it to Elasticsearch to handle it.

Reply:
Just for you to know, that not the only limit duplicated into Kuzzle. Actually the `documentsWriteCount` limit is also a duplicate of an ES limit. See [Threadpool.write](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-threadpool.html) 

If you upgrade this limit or send to many requests (like 300 call to `document.update` in a short period of time), you will hit the ES limit which is 200 by default:
`EsThreadPoolExecutor[name = alyx/write, queue capacity = 200, org.elasticsearch.common.util.concurrent.EsThreadPoolExecutor@59828a6b[Running, pool size = 2, active threads = 2, queued tasks = 200, completed tasks = 1326511]]`

Reply:
Good point. But is is really the same thing? It seems that the thread pool is more similar to our funnel request buffer than to a hard limit about the number of updates to perform.

If I'm right, then this is not exactly the same thing and we shouldn't worry about it.
But if I'm wrong, then we should at lest, during the ES service init, fetch the current Threadpool.write limit from ES, and fail to start Kuzzle if our WriteLimit is greater than this value.

What do you think?

Reply:
The thread pool is more similar to the funnel request buffer yes so we don't need to read this value from ES. But I think we could use a better error message